lib/core/src/Cardano/Wallet/DB/Store/CBOR/Store.hs

```diff
@@ -0,0 +1,82 @@
+
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ViewPatterns #-}
```

Comment:
```suggestion
```
I find `ViewPatterns` extremely confusing — the order of arguments gets all messed up. 🙈